open-apparel-registry/open-apparel-registry
# Add affiliations and certifications fields to facility claim profiles

## Overview

Add affiliations and certifications fields to facility claim profiles. The fields are ArrayFields in Django and are chosen using react-select multiselects in the UI. In the facility details page we display the selections as lists.

Connects #654 
Connects #653 

## Demo

![Screen Shot 2019-07-12 at 1 51 15 PM](https://user-images.githubusercontent.com/4165523/61148466-da946800-a4ac-11e9-9277-fc099c402eab.png)
![Screen Shot 2019-07-12 at 1 51 19 PM](https://user-images.githubusercontent.com/4165523/61148468-da946800-a4ac-11e9-848f-ce740bc72f83.png)

## Testing Instructions

- serve branch, then `resetdb` && `processfixtures`
- claim a facility and then approve the claim
- on the claim profile form, select some affiliations and certifications options, then save
- verify that the save is persisted and that the affiliations and certs are visible in the UI
- verify that the affiliation and certification changes appear in the update email

## Checklist
